app/views/layouts/application.html.erb

```diff
@@ -2,11 +2,28 @@
 <html>
 <head>
   <title>FridgeOverflow</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= stylesheet_link_tag    'application', media: 'all' %>
+  <%= javascript_include_tag 'application' %>
   <%= csrf_meta_tags %>
 </head>
 <body>
+  <div id='header'>
+    <ul>
+        <li id='home-page-link'><%= link_to "Fridge Overflow", root_path %></li>
+      <% if logged_in? %>
```

Comment:
Move this to a partial. You don't want your layout file changing very often at all.